go.mod

```diff
@@ -33,4 +34,5 @@ require (
 	google.golang.org/api v0.0.0-20180404000327-3097bf831ede
 	google.golang.org/genproto v0.0.0-20180716172848-2731d4fa720b // indirect
 	google.golang.org/grpc v0.0.0-20180510165436-d07538b1475e
+	gopkg.in/yaml.v2 v2.2.1-20180328165020-5420a8b6744d // indirect
```

Comment:
This is invalid.  This script should work:

```
git checkout master -- go.mod
rm -f go.sum
go mod edit --require=github.com/ghodss/yaml@0ca9ea5df5451ffdf184b4428c902747c2c11cd7 --require=gopkg.in/yaml.v2@5420a8b6744d3b0345ab293f6fcba19c978f1183
go test ./kythe/...
```

It'll leave the yaml version floating, but that's probably fine for now.

Reply:
That's almost exactly what I did in 43ca'

https://github.com/kythe/kythe/pull/3219/commits/43ca290300c4a073dfb080f37c85de54d453f45f#diff-37aff102a57d3d7b797f152915a6dc16

I am starting to suspect I have another transitive dep missing, because the go test ./kythe/... step clobbers out the commit hash from go.mod

Reply:
I couldn't find such a missing dep yesterday, and the only other idea I had was maybe my branch started too far out of date.